polaris/__init__.py

```diff
@@ -4,7 +4,16 @@
 from loguru import logger
 
 from ._version import __version__
-from .loader import load_benchmark, load_dataset, load_competition
+from .loader import load_benchmark, load_competition, load_dataset
+
+try:
+    # Register imagecodecs if they are available.
+    from imagecodecs.numcodecs import register_codecs
+
+    register_codecs()
+except ImportError:
+    pass
```

Comment:
**Suggestion**
I'd put this in `polaris/dataset/zarr/_utils.py`, to be close the the `check_zarr_codecs` function. Otherwise, that's pretty much how you handle optional dependencies.